src/setup_payload/SetupPayload.h

```diff
@@ -72,15 +72,18 @@ const int kTotalPayloadDataSizeInBytes = kTotalPayloadDataSizeInBits / 8;
 
 const char * const kQRCodePrefix = "CH:";
 
+/**
+ * The rendezvous type this device supports.
+ */
 enum class RendezvousInformationFlags : uint16_t
 {
-    kNone     = 0,      // Device does not support any method for rendezvous
-    kSoftAP   = 1 << 0, // Device supports hosting a SoftAP
-    kBLE      = 1 << 1, // Device supports BLE
-    kThread   = 1 << 2, // Device supports Thread
-    kEthernet = 1 << 3, // Device MAY be attached to a wired 802." connection"
+    kNone     = 0,      //< Device does not support any method for rendezvous
+    kWiFi     = 1 << 0, //< Device supports Wi-Fi
```

Comment:
nit: `WiFi` instead of `Wi-Fi`

Reply:
Pedantically, the official Wi-Fi Alliance spelling is "Wi-Fi" FWIW. I think we should hew to the official spelling. Here, since "-" is an illegal character for C/C++, we either need to live with "kWiFi" or use "kWi_Fi".